# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: supply-chain/config.toml
# Commit: da557d1806dd
# Full Hash: da557d1806dd9e1e519e00a4481b2e075f722ae8
# Author: Butkovits Atila <abutkovits@mozilla.com>
# Date: 2023-01-27 22:54 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing bustages complaining about AVIFDecodedData. CLOSED TREE
# ==============================================================================

diff -r d980bebc6d26 -r da557d1806dd supply-chain/config.toml
--- a/supply-chain/config.toml	Fri Jan 27 20:35:34 2023 +0000
+++ b/supply-chain/config.toml	Fri Jan 27 22:54:08 2023 +0200
@@ -671,6 +671,10 @@
 version = "1.0.2"
 criteria = "safe-to-run"
 
+[[exemptions.humantime]]
+version = "2.1.0"
+criteria = "safe-to-deploy"
+
 [[exemptions.hyper]]
 version = "0.14.19"
 criteria = "safe-to-run"